db/migrate/20220630043942_add_name_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
```

Comment:
```suggestion
    add_column :users, :name, :string, null: false
```
also add validation in model for presence: true